src/App.js

```diff
@@ -28,8 +28,13 @@ import { CLIENT_DEFAULTS } from './utils/CLIENT_DEFAULTS';
 // LOCALIZATION
 import { getTextForLanguage } from './utils/getTextForLanguage';
 
+/** App component; main top-level component of the app */
```

Comment:
My intention was to identify the declaration of the class/component itself as distinct from the declaration of the constructor function within the class. Maybe what I put didn't do that super clearly (open to suggestions about how to be moreso, again pending discussion about whether we even include these basics in the docs) but my goal was to label each as what it is.

General JSDoc thing to note (I'm super new to JSDoc as well, but have been reading up): in JSDoc a documentation comment immediately precedes whatever it applies to. So whenever in doubt about what is being documented in a given comment, the answer ought to be whatever immediately follows the documentation comment.

Reply:
In general, JSDoc guidelines seem to recommend documenting both classes and methods within them (ie in this case the constructor function here). Obvi what we choose to do is up to us tho

Reply:
Some things that may be helpful to include:
- The note about `HashRouter` and gh-pages.
- It manages the Dev HUD too.
- It manages the header and footer that appear on every page except the home page.
- It sends in the initial client values.

Not sure all of that needs to be included, but those are some things we can discuss.

Reply:
These seem like potentially useful inclusions to me, enough so that I'll go ahead and include them at least for now so that the PR is reflecting a "current potential most promising" version as we discuss rather than just my initial version until we settle on some things in discussion. Further thoughts from anyone welcome though!

Reply:
Implemented in [2bfee53](https://github.com/skudbucket/cliff-effects/commit/2bfee53a7ea49dea0c13c6ac17387b4f37f135fa)